luke/index.js

```diff
@@ -0,0 +1,14 @@
+// Declare a variable: 'It's hard to say who is right in this matter'.
+let argument = `It's hard to say who is right in this matter`;
+
+// 0.  Find the first occurrence of the letter "a" in a string, starting the search at position 17.
+console.log(argument.slice(17).indexOf(`a`));
```

Comment:
The first occurrence of the letter "a" starting from 17 is not 22 , but 39 !
Here is the correct solution:
console.log(argument.indexOf("a", 17)); //39

Reply:
added changes thanks ciamac :)

Reply:
Thanks:) No prob:)